go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -10,23 +10,31 @@ import (
 	kapi "k8s.io/api/core/v1"
 )
 
-func (ovn *Controller) getLoadBalancer(protocol kapi.Protocol) string {
+func (ovn *Controller) getLoadBalancer(protocol kapi.Protocol) (string,
+	error) {
 	if outStr, ok := ovn.loadbalancerClusterCache[string(protocol)]; ok {
-		return outStr
+		return outStr, nil
 	}
 
 	var out string
+	var err error
 	if protocol == kapi.ProtocolTCP {
-		out, _, _ = util.RunOVNNbctlUnix("--data=bare",
+		out, _, err = util.RunOVNNbctlUnix("--data=bare",
```

Comment:
/lgtm
At some point we might want to introduce a check that this call returns only one uuid. Because of installation issues, I have seen situation where multiple load balancers have the external_id k8s-cluster-lb-tcp=yes.
Another PR obviously.